src/components/ChatLog.js

```diff
@@ -3,25 +3,25 @@ import './ChatLog.css';
 import ChatEntry from './ChatEntry';
 import PropTypes from 'prop-types';
 
-const ChatLog = (props) => {
-  const chatComponents = props.entries.map((entry, index) => {
+const ChatLog = ({ entries, onLikeClicked }) => {
+  const chatComponents = entries.map((entry, index) => {
     return (
-      <p key={index}>
+      <div key={index}>
         <ChatEntry
           id={entry.id}
           sender={entry.sender}
           body={entry.body}
           timeStamp={entry.timeStamp}
           liked={entry.liked}
-          onUpdate={props.onUpdateEntry}
+          onLikeClicked={() => onLikeClicked(entry.id)}
```

Comment:
we pass in the id here instead of in ChatEntry itself, so ChatEntry does not even have to care about that functionality either